prisma/prisma-cloud/blocks/article/article.js

```diff
@@ -1,74 +1,206 @@
 import {
-  renderSidenav,
-  getPlaceholders,
-  loadBook,
-  parseFragment,
   PATH_PREFIX,
-  render,
-  SPA_NAVIGATION,
   REDIRECTED_ARTICLE_KEY,
+  SPA_NAVIGATION,
   decorateMain,
+  getPlaceholders,
   loadArticle,
+  loadBook,
+  parseFragment,
+  render,
+  renderSidenav,
   setBranch,
 } from '../../scripts/scripts.js';
 
-import {
-  getMetadata,
-  loadBlocks,
-  updateSectionsStatus,
-} from '../../scripts/lib-franklin.js';
+import { getMetadata, loadBlocks, updateSectionsStatus } from '../../scripts/lib-franklin.js';
+
+import '../../scripts/scroll-spy.js';
```

Comment:
could this be dynamic imported after initial render? same goes for theme-toggle

Reply:
Sure, they are now imported post-render.